### PR TITLE
core: a couple of tidyups to synthesized units

### PR DESCRIPTION
**Pull request by sourcejedi**
specifically the units

* `init.scope`
* `-.slice`
* and now `system.slice`; see description of second commit

### CODE REVIEW

**Comment by sourcejedi** [comment]
re commit 2, I notice we were logging

```
Feb 19 13:29:00 alan-laptop systemd[1]: Created slice System Slice.
Feb 19 13:29:00 alan-laptop systemd[1]: Reached target Slices.
```

sometime _after_ the initial activation of `-.mount`.  (Which is not logged like this; I don't think any of the perpetual units are).

File: src/core/scope.c [hunk 1]
```diff
@@ -163,9 +163,6 @@ static int scope_load_init_scope(Unit *u) {
          * synthesize it here, instead of relying on the unit file on disk. */
 
         u->default_dependencies = false;
-        u->ignore_on_isolate = true;
-
-        SCOPE(u)->kill_context.kill_signal = SIGRTMIN+14;
```
Comment (poettering) [inline src/core/scope.c:168]: hmm, but, but, but, this was so pretty! ;-)

**Comment by poettering** [comment]
btw, i very much appreciate the love you are giving all this. It's awesome having somebody review this in its current state and actually think about this!

**Comment by keszybz** [comment]
```
../src/firstboot/firstboot.c: In function 'process_root_password':
../src/firstboot/firstboot.c:670:24: error: implicit declaration of function 'crypt'; did you mean 'creat'? [-Werror=implicit-function-declaration]
         item.sp_pwdp = crypt(arg_root_password, salt);
                        ^~~~~
                        creat
```
Hmmm.

**Pull request merged**